pandas/tests/sparse/test_frame.py

```diff
@@ -1118,6 +1120,28 @@ def test_isnotnull(self):
                             'B': [True, False, True, True, False]})
         tm.assert_frame_equal(res.to_dense(), exp)
 
+    @pytest.mark.parameterize('index', [None, list('abc')])
+    @pytest.mark.parameterize('columns', [None, list('def')])
+    @pytest.mark.parameterize('fill_value', [None, 0, np.nan])
+    def test_from_to_scipy(self, index=None, columns=None, fill_value=None):
+        # GH 4343
+        tm._skip_if_no_scipy()
+        from scipy.sparse import csr_matrix
```

Comment:
can you also make a test with exercises *all* of the scipy input matrix types? (you may want to make these as fixtures for convenience)